pkg/asset/installconfig/powervs/platform.go

```diff
@@ -0,0 +1,31 @@
+package powervs
+
+import (
+	"os"
+
+	"github.com/openshift/installer/pkg/types/powervs"
+)
+
+// Platform collects powervs-specific configuration.
+func Platform() (*powervs.Platform, error) {
+
+	ssn, err := GetSession()
+	if err != nil {
+		return nil, err
+	}
+
+	var p powervs.Platform
+
+	// @TODO: The way we're using this (a precreated boot image in a Power VS Service instance) doesn't
+	// align with the installer's definition of this. We need a new var here and in the install config.
+	// This should be done before code cutoff in a followon PR.
+	if osOverride := os.Getenv("OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE"); len(osOverride) != 0 {
```

Comment:
the way to override would be to create the manifests and just change the `image` reference to point to the pre-created bootimage right? this wouldn't be needed

Reply:
that's one thing we could do that hadn't been discussed before, yah. but i would still like to have some kind of var that points to an ova and still have the installer create the boot image so that the user doesn't have to both put it in a cos bucket, *and* import it into the appropriate service id. also, we're planning to create the power-iass service instance on the fly, so have a pre-created boot image to reference wouldn't be an option. and that, iirc, is what you said we shouldn't also use this `OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE` for.

Reply:
right..an extra step of importing it into the service instance compared to other clouds but i prefer that than the approach of specifying the env variable to the override env. i'm not entirely opposed to this..but just questioning the need and making sure we are consistent with other clouds

Reply:
I am fine leaving this in ATM but it will need to be removed before feature freeze. Users should never use `OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE`. If you want users to be able to tweak it, it should go in the install config. OpenStack has a pattern with an optional ClusterOSImage if you want to check that out.